src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java

```diff
@@ -27,6 +27,13 @@ public UrlBuilder(String env, String browser) {
     this.browser = browser;
   }
 
+  public String getUrlForPage(Page page) {
+    if (page.getWikiPath() == null) {
+      return getUrlForWiki(page.getWikiName());
+    }
+    return getUrlForPath(page.getWikiName(), page.getWikiPath());
+  }
+
   public String getUrlForPath(String wikiName, String wikiPath) {
     String url = getUrlForWiki(wikiName);
     String separator = wikiName.endsWith("wikia") || wikiName.equals("wowwiki") ? "" : "wiki/";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wikia" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "wowwiki" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1132)